WL-4031 Added spinning wheel when Remove Participant group is clicked.

ouit0408 opened the pull request:
Added spinny wheel image for remove participant  button in 'chef_site_siteInfo-list.vm'

buckett commented on site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-list.vm:
```diff
@@ -523,10 +527,13 @@ function printPreview(target){
 								<input type="hidden" name="sakai_csrf_token" value="$sakai_csrf_token" />
 								<input type="submit" name="eventSubmit_doRemoveGroup" value="$tlang.getString("sitegen.siteinfolist.remove")"/>
+								<span id="spinnerRemove" style="display:none;">
+									<img class="loader" src="/../../library/newspicker/css/images/ajax-loader.gif" alt="Loading"/>
```
Much better to just have `/library/...` as it the URL of the page includes more parts the link will break.

buckett commented on site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-list.vm:
```diff
@@ -523,10 +527,13 @@ function printPreview(target){
 						<td>
 							#if ($allowUpdateSiteMembership)
-							<form name="removeGroupForm" class="inlineForm" method="post" action="#toolForm("SiteAction")">
+							<form id="removeGroupForm" name="removeGroupForm" class="inlineForm" method="post" action="#toolForm("SiteAction")">
```
You shouldn't have multiple elements in the same page with the same ID (as this is in a loop).

buckett commented:
Still waiting for response to comments.

buckett commented:
Still has comments outstanding.

ouit0408 commented:
sorry I missed the previous comments , will have a look soon.

ouit0408 committed:
WL-4031 Added spinning wheel when Remove Participant group is clicked.

Using form name to capture the submit event and then span class to display
the spinner.

ouit0408 commented:
Implemented the review comments and have pushed the code.
